src/wizard/interactive_build.jl

```diff
@@ -503,7 +505,7 @@ function step5c(state::WizardState)
                 verbose=false,
                 tee_stream=state.outs
             )
-
+            prefix == nothing && (skip = false; return)
```

Comment:
`skip = true`?

Reply:
oh... I guess the return was enough to skip then, since it was actually working for me!

Reply:
I think `ok = false` is also appropriate here, rather than adding a new `skip` flag.

Reply:
That'd be great - I was expecting setting ok to false to interrupt the build... If that _officially_ skips the build, then this could almost be a good solution ;)

Reply:
If that actually goes into a mergeable direction, we should also improve the try catch!